.nvmrc~

```diff
@@ -1 +1 @@
-lts/dubnium
+lts/hydrogen
```

Comment:
As a quick note, these files that ends with `~` are just safe copies generated by editor. I should incorrectly left it the time that i edit this project. 

Also this will not be effective to change the node engine version by `nvm`

Reply:
> As a quick note, these files that ends with `~` are just safe copies generated by editor. I should incorrectly left it the time that i edit this project.
> 
> Also this will not be effective to change the node engine version by `nvm`

@ValentinaForeroConde as @fasozu mentioned in this comment, let's remove this file from the code in another PR. :)